config/reaction-oss/reaction-v3.4.0.mk

```diff
@@ -0,0 +1,34 @@
+###############################################################################
```

Comment:
This is what I was trying to prevent with my comments on #114, @ticean. We've been making a new file in this folder every time any of the child projects change. In a year this folder could have hundreds of files in it, even though the same versioning is accessible via git tags. This PR is incrementing the version of this very repository to `v3.4.0` (as shown in the changelog changes above).

I know we discussed only putting out a new release of this when something noteworthy changed. Where did we fall on that? The pushback I gave at the time was that we could just have one reaction-oss.mk file and keep editing it which is certainly an extreme, but I think of this as the other extreme. Is everyone else okay with this after seeing it in action, or should we revisit this decision?

Given that we release a new file every time we increment the version, my preference is still to rely on git tags and one file. Checking out tag `v3.3.0` of this repo would let you use the `reaction-oss.mk` file at that version. If I'm still alone on this, I at least think we should change how we version this repo. What happens if a makefile gets inserted in `local` or another sibling folder to `reaction-oss`? The versioning breaks down, and next time we add a versioned file in this `reaction-oss` folder there will be a gap in the file names. Or we would copy the latest file and call it the latest version, just for consistency. I do not like either of these options.

Reply:
Maybe we don’t release this project anymore, and we just have ‘trunk’ as the latest from all projects, and if you want to run an older version you just update the config to match whatever versions you want?

You’d have to update the config anyway if you want the newest of one project, and an older version of a different project, it would be easier to just say “this is the latest, you can change versions if you want” instead of “pick the config that matches exactly what you want”

Reply:
I think that we should go ahead and release this one as-is so that we don't block the release. This is technically following the patterns that we've been using and adopted with the merge of #114. 

That said, @rosshadden is raising a good discussion and we should continue with it, but maybe we can do that in a non-blocking venue. Ross and I discussed this in a zoom and I think we've got more agreement than differences, but there are details to work through. The discussion has a lot of touch on our general release process.

@rosshadden if you agree, want to open a dedicated issue for this and copy your comment into it?